core/rawdb/database.go

```diff
@@ -451,7 +452,8 @@ func (s *stat) Count() string {
 
 // InspectDatabase traverses the entire database and checks the size
 // of all different categories of data.
-func InspectDatabase(db ethdb.Database, keyPrefix, keyStart []byte) error {
+func InspectDatabase(db ethdb.Database, keyPrefix, keyStart []byte, opts ...InspectDatabaseOption) error {
```

Comment:
It might be interesting to write a test/example-test using `InspectDatabase` with options. Aka this really feels like we need a test for InspectDatabase (both here and upstream)

Reply:
An example would be great because it also avoids having to change the `os.Stdout` destination to make testing possible.

Reply:
I can work on this, let me know 😉 And yes an example fits perfectly (and we can PR upstream as well)